### PR TITLE
Added Image Processing #6500

### LINKED ISSUE #6500: [Python] : Image Processing using Python
### Description

- What is Image Processing?
- How Python can be used for image processing?
- Examples

### Domain

Python_Domain

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by burpcat**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #6500 

- Learnt and understood the Image Processing part of python using pillow library
- Understood and Implemented the same using Jupyter Notebook

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [x] Python_Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [ x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [ x] I have performed a self-review of my own code or work.
- [ x] I have commented my code, particularly in hard-to-understand areas.
- [ x] My changes generates no new warnings.
- [ x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by burpcat** [comment]
@adityabisoi , could you please review my PR

File: Python/Image_Processing/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+# #6500 [Python] : Image Processing using Python
+
+- Image Processing is the method through which, some meaningful operations can be performed on the Image using tools or languages.
+
+- We use Python here for the Image processing here
+
+- Contributed by [burpcat](https://www.github.com/burpcat)
```
Comment (queenish001) [inline Python/Image_Processing/README.md:8]: 1. Please remove this line, Contributors are not supposed to add their credentials in their contributions.
2. Make the README more elaborate regarding the topic.

**Review by queenish001 — CHANGES_REQUESTED** [review]
- Remove the `Contributed by` line from your Jupyter Notebook as well.
- Remove the `.DS_Store` file that you've uploaded

File: Python/Image_Processing/README.md (hunk 1 above)
Comment (burpcat) [reply Python/Image_Processing/README.md]: I made the changes, please review

**Review by queenish001 — APPROVED** [review]
LGTM 👍 

Please Merge this @adityabisoi

**Pull request merged**